rd_rw612_bga/frdm_rw612: wifi: shell: Failing to build with disable 802.11ax

<!--
**Notes**
Github Discussions (https://github.com/zephyrproject-rtos/zephyr/discussions)
are available to first verify that the issue is a genuine Zephyr bug and not a
consequence of Zephyr services misuse.

This issue list is only for bugs in the main Zephyr code base
(https://github.com/zephyrproject-rtos/zephyr/). If the bug is for a project
fork (such as NCS) specific feature, please open an issue in the fork project
instead.
-->

**Describe the bug**
<!--
A clear and concise description of what the bug is.

Please also mention any information which could help others to understand
the problem you're facing:
 - What target platform are you using?
 - What have you tried to diagnose or workaround this issue?
 - Is this a regression? If yes, have you been able to "git bisect" it to a
   specific commit?
 - ...
-->

Building for rd_rw612_bga/frdm_rw612 fails when disabling 802.11ax.

**To Reproduce**
<!--
Steps to reproduce the behavior:
1. mkdir build; cd build
2. cmake -DBOARD=board\_xyz
3. make
4. See error
-->
1. `cd ~/zephyrproject/zephyr/`
2. `west update` as needed
3. `west build -b rd_rw612_bga samples/net/wifi/shell -d build/wifi/ -- -DCONFIG_NXP_WIFI_11AX=n`
  or  `west build -b frdm_rw612 samples/net/wifi/shell -d build/wifi/ -- -DCONFIG_NXP_WIFI_11AX=n`


**Expected behavior**
<!--
A clear and concise description of what you expected to happen.
-->
Build should work. 
As it does e.g. for the `rd_rw612_bga`: `west build -b rd_rw612_bga samples/net/wifi/shell -d build/wifi/ -- -DCONFIG_NXP_WIFI_11AX=n`

**Impact**
<!--
What impact does this issue have on your progress (e.g., annoyance, showstopper)
-->
Unable to disable 802.11ax functionality.

**Logs and console output**
<!--
If applicable, add console logs or other types of debug information
e.g Wireshark capture or Logic analyzer capture (upload in zip archive).
copy-and-paste text and put a code fence (\`\`\`) before and after, to help
explain the issue. (if unable to obtain text log, add a screenshot)
-->
```
west build -b rd_rw612_bga samples/net/wifi/shell -d build/wifi/ -- -DCONFIG_NXP_WIFI_11AX=n
-- west build: generating a build system
Loading Zephyr default modules (Zephyr base (cached)).
-- Application: /home/zephyrproject/zephyr/samples/net/wifi/shell
-- CMake version: 3.30.2
-- Cache files will be written to: /home/.cache/zephyr
-- Zephyr version: 4.0.0-rc2 (/home/zephyrproject/zephyr)
-- Found west (found suitable version "1.3.0", minimum required is "0.14.0")
-- Board: rd_rw612_bga, qualifiers: rw612
-- Found host-tools: zephyr 0.16.3 (/home/zephyr-sdk-0.16.3)
-- Found toolchain: zephyr 0.16.3 (/home/zephyr-sdk-0.16.3)
-- Found BOARD.dts: /home/zephyrproject/zephyr/boards/nxp/rd_rw612_bga/rd_rw612_bga.dts
-- Generated zephyr.dts: /home/zephyrproject/zephyr/build/wifi/zephyr/zephyr.dts
-- Generated pickled edt: /home/zephyrproject/zephyr/build/wifi/zephyr/edt.pickle
-- Generated zephyr.dts: /home/zephyrproject/zephyr/build/wifi/zephyr/zephyr.dts
-- Generated devicetree_generated.h: /home/zephyrproject/zephyr/build/wifi/zephyr/include/generated/zephyr/devicetree_generated.h
-- Including generated dts.cmake file: /home/zephyrproject/zephyr/build/wifi/zephyr/dts.cmake
Parsing /home/zephyrproject/zephyr/Kconfig
Loaded configuration '/home/zephyrproject/zephyr/boards/nxp/rd_rw612_bga/rd_rw612_bga_defconfig'
Merged configuration '/home/zephyrproject/zephyr/samples/net/wifi/shell/prj.conf'
Merged configuration '/home/zephyrproject/zephyr/samples/net/wifi/shell/boards/rd_rw612_bga.conf'
Merged configuration '/home/zephyrproject/zephyr/build/wifi/zephyr/misc/generated/extra_kconfig_options.conf'
No change to configuration in '/home/zephyrproject/zephyr/build/wifi/zephyr/.config'
No change to Kconfig header in '/home/zephyrproject/zephyr/build/wifi/zephyr/include/generated/zephyr/autoconf.h'
Load components for RW612:
driver_common component is included.
driver_reset component is included.
device_CMSIS component is included.
CMSIS_Include_core_cm component is included.
device_startup component is included.
driver_lpc_gpio component is included.
driver_pint component is included.
driver_inputmux component is included.
driver_inputmux_connections component is included.
driver_ostimer component is included.
driver_flexcomm component is included.
driver_flexcomm_usart component is included.
driver_trng component is included.
driver_imu component is included.
driver_cache_cache64 component is included.
component_mflash_rdrw610 component is included.
component_mflash_common component is included.
driver_flexspi component is included.
component_osa_zephyr component is included from /home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/components/osa/set_component_osa.cmake.
component_wifi_bt_module_tx_pwr_limits component is included.
driver_conn_fwloader component is included.
driver_ocotp component is included.
component_wireless_imu_adapter component is included.
driver_gdma component is included.
--  generate include of binary blob: /home/zephyrproject/zephyr/modules/hal/nxp/zephyr/blobs/rw61x/rw61x_sb_wifi_a2.bin
CMake Warning at /home/zephyrproject/zephyr/modules/hostap/CMakeLists.txt:670 (message):
  CONFIG_SAE_PWE_EARLY_EXIT is enabled, this is not secure and is a
  workaround for low resource systems, please use it carefully and do not use
  it production.


-- Using ccache: /usr/bin/ccache
CMake Warning at /home/zephyrproject/zephyr/CMakeLists.txt:2139 (message):
  __ASSERT() statements are globally ENABLED


-- Configuring done (3.4s)
-- Generating done (0.2s)
-- Build files have been written to: /home/zephyrproject/zephyr/build/wifi
-- west build: building application
[429/659] Building C object zephyr/drivers/wifi/CMakeFiles/drivers__wifi.dir/nxp/nxp_wifi_drv.c.obj
FAILED: zephyr/drivers/wifi/CMakeFiles/drivers__wifi.dir/nxp/nxp_wifi_drv.c.obj
ccache /home/zephyr-sdk-0.16.3/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DBOARD_FLASH_SIZE="CONFIG_FLASH_SIZE*1024" -DBOOT_HEADER_ENABLE=1 -DCPU_RW612ETA2I -DKERNEL -DK_HEAP_MEM_POOL_SIZE=122880 -DPICOLIBC_DOUBLE_PRINTF_SCANF -DRW610 -DWIFI_BT_TX_PWR_LIMITS=\"wlan_txpwrlimit_cfg_WW_rw610.h\" -D__LINUX_ERRNO_EXTENSIONS__ -D__PROGRAM_START -D__ZEPHYR_SUPERVISOR__ -D__ZEPHYR__=1 -I/home/zephyrproject/zephyr/build/wifi/zephyr/include/generated/zephyr -I/home/zephyrproject/zephyr/include -I/home/zephyrproject/zephyr/build/wifi/zephyr/include/generated -I/home/zephyrproject/zephyr/soc/nxp/rw -I/home/zephyrproject/zephyr/include/zephyr/posix -I/home/zephyrproject/zephyr/lib/posix/options/getopt -I/home/zephyrproject/zephyr/soc/nxp/rw/. -I/home/zephyrproject/zephyr/subsys/net/l2 -I/home/zephyrproject/zephyr/build/wifi/zephyr/misc/generated -I/home/zephyrproject/zephyr/subsys/net/lib/dns/. -I/home/zephyrproject/zephyr/subsys/shell/modules/kernel_service/thread/.. -I/home/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/home/zephyrproject/zephyr/modules/cmsis/. -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/devices/RW612 -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/devices/RW612/drivers -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/incl -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/incl/wifidriver -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/incl/wlcmgr -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/incl/port/osa -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/incl/port/net/zephyr -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/port/osa -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/port/net/zephyr -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/sdio_nxp_abs/incl -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/firmware_dnld -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/fwdnld_intf_abs -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/wifidriver -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/wifidriver/incl -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/wifi_bt_firmware -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/wifi_bt_firmware/8987 -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/wifi_bt_firmware/IW416 -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/wifi_bt_firmware/nw61x -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/wifi_bt_firmware/8801 -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/cli -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/components/osa -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/components/lists -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/components/wifi_bt_module/AzureWave/tx_pwr_limits -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/components/wifi_bt_module/Murata/tx_pwr_limits -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/components/rpmsg -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/certs -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/wifidriver/wpa_supp_if/incl -I/home/zephyrproject/modules/hal/nxp/mcux/middleware/wifi_nxp/incl/port/mbedtls -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/common -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/lpc_gpio -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/pint -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/inputmux -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/ostimer -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/flexcomm -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/flexcomm/usart -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/trng -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/imu -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/cache/cache64 -I/home/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/components/osa -I/home/zephyrproject/zephyr/modules/hal_nxp/. -I/home/zephyrproject/zephyr/modules/hostap/src -I/home/zephyrproject/modules/lib/hostap -I/home/zephyrproject/modules/lib/hostap/wpa_supplicant -I/home/zephyrproject/modules/lib/hostap/src -I/home/zephyrproject/modules/lib/hostap/src/common -I/home/zephyrproject/modules/lib/hostap/src/eap_common -I/home/zephyrproject/modules/lib/hostap/src/eap_server -I/home/zephyrproject/modules/lib/hostap/src/radius -I/home/zephyrproject/modules/lib/hostap/src/crypto -I/home/zephyrproject/modules/lib/hostap/src/ap -I/home/zephyrproject/modules/lib/hostap/src/drivers -I/home/zephyrproject/modules/lib/hostap/src/rsn_supp -I/home/zephyrproject/modules/lib/hostap/hostapd -I/home/zephyrproject/modules/lib/hostap/port/mbedtls -I/home/zephyrproject/modules/lib/hostap/src/utils -isystem /home/zephyrproject/zephyr/lib/libc/common/include -Wshadow -fno-strict-aliasing -O2 -imacros /home/zephyrproject/zephyr/build/wifi/zephyr/include/generated/zephyr/autoconf.h -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m33+nodsp -mthumb -mabi=aapcs -mfp16-format=ieee -mtp=soft --sysroot=/home/zephyr-sdk-0.16.3/arm-zephyr-eabi/arm-zephyr-eabi -imacros /home/zephyrproject/zephyr/include/zephyr/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wdouble-promotion -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-pic -fno-pie -fno-asynchronous-unwind-tables -ftls-model=local-exec -fno-reorder-functions --param=min-pagesize=0 -fno-defer-pop -fmacro-prefix-map=/home/zephyrproject/zephyr/samples/net/wifi/shell=CMAKE_SOURCE_DIR -fmacro-prefix-map=/home/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/home/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -mcmse -specs=picolibc.specs -D_POSIX_THREADS -std=c99 -MD -MT zephyr/drivers/wifi/CMakeFiles/drivers__wifi.dir/nxp/nxp_wifi_drv.c.obj -MF zephyr/drivers/wifi/CMakeFiles/drivers__wifi.dir/nxp/nxp_wifi_drv.c.obj.d -o zephyr/drivers/wifi/CMakeFiles/drivers__wifi.dir/nxp/nxp_wifi_drv.c.obj -c /home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c: In function 'nxp_wifi_wlan_event_callback':
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:284:25: error: 'else' without a previous 'if'
  284 |                         else if (nxp_wlan_network.dot11ac) {
      |                         ^~~~
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c: In function 'nxp_wifi_set_twt':
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1340:9: error: unknown type name 'wlan_twt_setup_config_t'
 1340 |         wlan_twt_setup_config_t twt_setup_conf;
      |         ^~~~~~~~~~~~~~~~~~~~~~~
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1341:9: error: unknown type name 'wlan_twt_teardown_config_t'
 1341 |         wlan_twt_teardown_config_t teardown_conf;
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1351:31: error: request for member 'implicit' in something not a structure or union
 1351 |                 twt_setup_conf.implicit = params->setup.implicit;
      |                               ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1352:31: error: request for member 'announced' in something not a structure or union
 1352 |                 twt_setup_conf.announced = params->setup.announce;
      |                               ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1353:31: error: request for member 'trigger_enabled' in something not a structure or union
 1353 |                 twt_setup_conf.trigger_enabled = params->setup.trigger;
      |                               ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1354:31: error: request for member 'negotiation_type' in something not a structure or union
 1354 |                 twt_setup_conf.negotiation_type = params->negotiation_type;
      |                               ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1355:31: error: request for member 'twt_wakeup_duration' in something not a structure or union
 1355 |                 twt_setup_conf.twt_wakeup_duration = params->setup.twt_wake_interval;
      |                               ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1356:31: error: request for member 'flow_identifier' in something not a structure or union
 1356 |                 twt_setup_conf.flow_identifier = params->flow_id;
      |                               ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1357:31: error: request for member 'hard_constraint' in something not a structure or union
 1357 |                 twt_setup_conf.hard_constraint = 1;
      |                               ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1358:31: error: request for member 'twt_mantissa' in something not a structure or union
 1358 |                 twt_setup_conf.twt_mantissa = params->setup.twt_interval;
      |                               ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1359:31: error: request for member 'twt_request' in something not a structure or union
 1359 |                 twt_setup_conf.twt_request = params->setup.responder;
      |                               ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1360:23: warning: implicit declaration of function 'wlan_set_twt_setup_cfg'; did you mean 'wlan_set_tsp_cfg'? [-Wimplicit-function-declaration]
 1360 |                 ret = wlan_set_twt_setup_cfg(&twt_setup_conf);
      |                       ^~~~~~~~~~~~~~~~~~~~~~
      |                       wlan_set_tsp_cfg
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1362:30: error: request for member 'flow_identifier' in something not a structure or union
 1362 |                 teardown_conf.flow_identifier = params->flow_id;
      |                              ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1363:30: error: request for member 'negotiation_type' in something not a structure or union
 1363 |                 teardown_conf.negotiation_type = params->negotiation_type;
      |                              ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1364:30: error: request for member 'teardown_all_twt' in something not a structure or union
 1364 |                 teardown_conf.teardown_all_twt = params->teardown.teardown_all;
      |                              ^
/home/zephyrproject/zephyr/drivers/wifi/nxp/nxp_wifi_drv.c:1365:23: warning: implicit declaration of function 'wlan_set_twt_teardown_cfg' [-Wimplicit-function-declaration]
 1365 |                 ret = wlan_set_twt_teardown_cfg(&teardown_conf);
      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~
[434/659] Building C object modules/hal_nxp/hal_nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/middleware/wifi_nxp/wifidriver/mlan_11k.c.obj
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: /usr/bin/cmake --build /home/zephyrproject/zephyr/build/wifi
```
**Environment (please complete the following information):**

 - OS: Linux
 - Toolchain: Zephyr SDK 0.16.3
 - Commit: 67980643d799f76aff93118346a8dabfb472a642

**Additional context**
<!--
Add any other context that could be relevant to your issue, such as pin setting,
target configuration, ...
-->
- The error comes from this line of code:
https://github.com/zephyrproject-rtos/zephyr/blob/67980643d799f76aff93118346a8dabfb472a642/drivers/wifi/nxp/nxp_wifi_drv.c#L284
   - when disabling 802.11ax (set CONFIG_NXP_WIFI_11AX=n), if-else does not match.

--- a/drivers/wifi/nxp/nxp_wifi_drv.c
+++ b/drivers/wifi/nxp/nxp_wifi_drv.c
@@ -276,20 +276,20 @@ int nxp_wifi_wlan_event_callback(enum wlan_event_reason reason, void *data)
 	case WLAN_REASON_UAP_CLIENT_CONN:
 		wlan_get_current_uap_network(&nxp_wlan_network);
 #ifdef CONFIG_NXP_WIFI_11AX
-			if (nxp_wlan_network.dot11ax) {
-				ap_sta_info.link_mode = WIFI_6;
-			}
+		if (nxp_wlan_network.dot11ax) {
+			ap_sta_info.link_mode = WIFI_6;
+		} else
 #endif
 #ifdef CONFIG_NXP_WIFI_11AC
-			else if (nxp_wlan_network.dot11ac) {
+			if (nxp_wlan_network.dot11ac) {
 				ap_sta_info.link_mode = WIFI_5;
-			}
+		} else
 #endif
-			else if (nxp_wlan_network.dot11n) {
+			if (nxp_wlan_network.dot11n) {
 				ap_sta_info.link_mode = WIFI_4;
-			} else {
-				ap_sta_info.link_mode = WIFI_3;
-			}
+		} else {
+			ap_sta_info.link_mode = WIFI_3;
+		}
 
 		memcpy(ap_sta_info.mac, data, WIFI_MAC_ADDR_LEN);
 		ap_sta_info.mac_length  = WIFI_MAC_ADDR_LEN;
@@ -968,15 +968,15 @@ static int nxp_wifi_status(const struct device *dev, struct wifi_iface_status *s
 #ifdef CONFIG_NXP_WIFI_11AX
 			if (nxp_wlan_network.dot11ax) {
 				status->link_mode = WIFI_6;
-			}
+			} else
 #endif
 #ifdef CONFIG_NXP_WIFI_11AC
-			else if (nxp_wlan_network.dot11ac) {
-				status->link_mode = WIFI_5;
-			}
+				if (nxp_wlan_network.dot11ac) {
+					status->link_mode = WIFI_5;
+			} else
 #endif
-			else if (nxp_wlan_network.dot11n) {
-				status->link_mode = WIFI_4;
+				if (nxp_wlan_network.dot11n) {
+					status->link_mode = WIFI_4;
 			} else {
 				status->link_mode = WIFI_3;
 			}
@@ -1335,6 +1335,7 @@ static int nxp_wifi_reg_domain(const struct device *dev, struct wifi_reg_domain 
 	return 0;
 }
 
+#ifdef CONFIG_NXP_WIFI_11AX_TWT
 static int nxp_wifi_set_twt(const struct device *dev, struct wifi_twt_params *params)
 {
 	wlan_twt_setup_config_t twt_setup_conf;
@@ -1367,6 +1368,7 @@ static int nxp_wifi_set_twt(const struct device *dev, struct wifi_twt_params *pa
 
 	return ret;
 }
+#endif
 
 static void nxp_wifi_sta_init(struct net_if *iface)
 {
@@ -1654,7 +1656,9 @@ static const struct wifi_mgmt_ops nxp_wifi_sta_mgmt = {
 	.cfg_11k = nxp_wifi_11k_cfg,
 	.set_power_save = nxp_wifi_power_save,
 	.get_power_save_config = nxp_wifi_get_power_save,
+#ifdef CONFIG_NXP_WIFI_11AX_TWT
 	.set_twt = nxp_wifi_set_twt,
+#endif
 };
 
 #if defined(CONFIG_WIFI_NM_WPA_SUPPLICANT)

